truecase/TrueCaser.py

```diff
@@ -90,16 +90,20 @@ def get_score(self, prev_token, possible_token, next_token):
     def first_token_case(self, raw):
         return f'{raw[0].upper()}{raw[1:]}'
 
```

Comment:
instead of a flag, can you expose two functions?

```
def get_true_case(self, sentence, out_of_vocabulary_token_option="title"):
  tokens = self.tknzr.tokenize(sentence)
  result_tokens = this.get_true_case_from_tokens(tokens, out_of_vocabulary_token_option)
  return "".join([" " + i if not i.startswith("'") and i not in string.punctuation else i for i in tokens_true_case]).strip()

def get_true_case_from_tokens(self, tokens, out_of_vocabulary_token_option="title"):
  ...
```

Then you can use get_true_case_from_tokens when you want to already have tokens, what do you think?

Reply:
Sure, that also works!

Reply:
@daltonfury42 sorry for the delay. I haven't had a chance to come back to this just yet. I can try to get to this in the next week or two. How does that sound?

Reply:
Hey. That works, I am passively maintaining the project, so, take your time. 

Thanks.